libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tab.component.scss

```diff
@@ -1,35 +1,184 @@
 @use 'libs/components/theme/src/lib/styles/variables' as *;
+@use 'libs/components/theme/src/lib/styles/compat-tokens-mixins' as compatMixins;
+
+@include compatMixins.sky-default-overrides('.sky-vertical-tab') {
+  --sky-override-vertical-tab-background-color: transparent;
+  --sky-override-vertical-tab-background-color-active: #{$sky-background-color-neutral-light};
+  --sky-override-vertical-tab-background-color-focus: transparent;
+  --sky-override-vertical-tab-background-color-hover: #{$sky-background-color-neutral-light};
+  --sky-override-vertical-tab-background-color-selected: #{$sky-background-color-neutral-light};
+  --sky-override-vertical-tab-border-left-active: none;
+  --sky-override-vertical-tab-border-left-hover: none;
+  --sky-override-vertical-tab-border-left-selected: #{$sky-nav-selected-border-width}
+    solid #{$sky-highlight-color-info};
+  --sky-override-vertical-tab-box-shadow-focus: none;
+  --sky-override-vertical-tab-count-margin-left: #{$sky-margin-half};
+  --sky-override-vertical-tab-font-color: #{$sky-text-color-default};
+  --sky-override-vertical-tab-font-color-disabled: var(
+    --sky-text-color-deemphasized
+  );
+  --sky-override-vertical-tab-font-selected-color: #{$sky-text-color-default};
+  --sky-override-vertical-tab-gap: #{$sky-margin-half};
+  --sky-override-vertical-tab-heading-error-padding-left: #{$sky-padding-half};
+  --sky-override-vertical-tab-hover-font-color: #{$sky-text-color-default};
+  --sky-override-vertical-tab-margin: #{$sky-margin-half} 0 #{$sky-margin-half}
+    #{$sky-margin};
+  --sky-override-vertical-tab-outline-focus: auto;
+  --sky-override-vertical-tab-padding: #{$sky-padding-3_4} 0 #{$sky-padding-3_4}
+    #{$sky-padding};
+  --sky-override-vertical-tab-padding-left-active: #{$sky-padding};
+  --sky-override-vertical-tab-padding-left-selected: 6px;
+  --sky-override-vertical-tab-padding-left-hover: #{$sky-padding};
+}
+
+@include compatMixins.sky-modern-overrides('.sky-vertical-tab') {
+  --sky-override-vertical-tab-background-color-disabled: transparent;
+  --sky-override-vertical-tab-background-color-selected: transparent;
+  --sky-override-vertical-tab-border-left-active: 4px solid
+    var(--modern-color-blue-50);
+  --sky-override-vertical-tab-border-left-hover: 1px solid
+    var(--modern-color-blue-50);
+  --sky-override-vertical-tab-gap: #{$sky-theme-modern-margin-inline-md};
+  --sky-override-vertical-tab-heading-error-padding-left: #{$sky-padding-half};
+  --sky-override-vertical-tab-margin: #{$sky-theme-modern-margin-inline-md} 0 #{$sky-theme-modern-margin-inline-md}
+    #{$sky-theme-modern-margin-inline-sm};
+  --sky-override-vertical-tab-content-pane-margin-bottom: 0;
+}
+
+:host {
+  &:has(~ sky-vertical-tab) .sky-vertical-tab {
+    margin-bottom: var(
+      --sky-override-vertical-tab-gap,
+      var(--sky-space-gap-action_group-l)
+    );
+  }
+
+  &:has(.sky-vertical-tabset-button-disabled) {
+    cursor: not-allowed;
```

Comment:
The previous rules for `not-allowed` were not working - this fixes that. Matt said he was fine with all themes getting this fix.